infra.bs

```diff
@@ -1042,6 +1042,19 @@ as 200/`<code>OK</code>`.
   <p class=note>The conventions used in this step are those of the JavaScript specification.
 </ol>
 
+<p>To <dfn export>JSON stringify and UTF-8 encode to bytes</dfn> a given JavaScript value
```

Comment:
Maybe omit "and UTF-8 encode" for symmetry? Since we assume everything is UTF-8 these days? Or maybe we should keep it.

Reply:
I personally would prefer it to be explicit.  but y'all are editors of this spec, so up to you.

Reply:
I personally would prefer it to be explicit.  but y'all are editors of this spec, so up to you. IIRC that is the phrase that Anne suggested a while back.